sdk/core/System.ServiceModel.Rest/api/System.ServiceModel.Rest.net6.0.cs

```diff
@@ -182,47 +186,51 @@ public partial interface IPipelineEnumerator
-        public void Process(TMessage message, TOptions options, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { }
-        public abstract System.Threading.Tasks.ValueTask ProcessAsync(TMessage message, TOptions options);
-        public System.Threading.Tasks.ValueTask ProcessAsync(TMessage message, TOptions options, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { throw null; }
+        public abstract TMessage CreateMessage(TMessageOptions options);
+        public abstract void Process(TMessage message);
+        public void Process(TMessage message, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { }
+        public abstract System.Threading.Tasks.ValueTask ProcessAsync(TMessage message);
+        public System.Threading.Tasks.ValueTask ProcessAsync(TMessage message, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { throw null; }
     }
-    public abstract partial class Pipeline<TMessage, TOptions>
+    public abstract partial class Pipeline<TMessage, TMessageOptions>
     {
         protected Pipeline() { }
-        public abstract TMessage CreateMessage();
-        public abstract void Send(TMessage message, TOptions options);
-        public abstract System.Threading.Tasks.ValueTask SendAsync(TMessage message, TOptions options);
+        public abstract TMessage CreateMessage(TMessageOptions options);
+        public abstract void Send(TMessage message);
+        public abstract System.Threading.Tasks.ValueTask SendAsync(TMessage message);
     }
-    public partial class ResponseBufferingPolicy : System.ServiceModel.Rest.Core.Pipeline.IPipelinePolicy<System.ServiceModel.Rest.Core.PipelineMessage, System.ServiceModel.Rest.InvocationOptions>
+    public partial class ResponseBufferingPolicy : System.ServiceModel.Rest.Core.Pipeline.IPipelinePolicy<System.ServiceModel.Rest.Core.PipelineMessage>
     {
         public ResponseBufferingPolicy(System.TimeSpan networkTimeout) { }
-        public void Process(System.ServiceModel.Rest.Core.PipelineMessage message, System.ServiceModel.Rest.InvocationOptions options, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { }
-        public System.Threading.Tasks.ValueTask ProcessAsync(System.ServiceModel.Rest.Core.PipelineMessage message, System.ServiceModel.Rest.InvocationOptions options, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { throw null; }
+        public void Process(System.ServiceModel.Rest.Core.PipelineMessage message, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { }
+        public System.Threading.Tasks.ValueTask ProcessAsync(System.ServiceModel.Rest.Core.PipelineMessage message, System.ServiceModel.Rest.Core.Pipeline.IPipelineEnumerator pipeline) { throw null; }
+        public static void SetBufferResponse(System.ServiceModel.Rest.Core.PipelineMessage message, bool bufferResponse) { }
```

Comment:
Static accessor methods on the policy.  I feel like these belong better on the transport, but given that we share a transport across all Azure SDK clients, this is a bit tricky.